src/fsharp/fsi/fsi.fs

```diff
@@ -2383,6 +2383,10 @@ type FsiEvaluationSession (fsi: FsiEvaluationSessionHostConfig, argv:string[], i
                                   true, // long running: optimizeForMemory 
                                   currentDirectory,isInteractive=true, 
                                   isInvalidationSupported=false)
+#if NETSTANDARD1_5
+    do tcConfigB.useMonoResolution <- true
+    do tcConfigB.primaryAssembly <- PrimaryAssembly.DotNetCore
+#endif
```

Comment:
Just to note: This will actually disable the msbuild resolution code path. Therefore the changes below are not actually needed (at least for my case) and I never tested them. I just wanted to bring some attention there as there seems to be a problem which should be fixed...